### PR TITLE
support missing information like channel_id, included-pdu_id and conn…

### PR DESCRIPTION
**Pull request by rawiamoalla**
…ector_id

### CODE REVIEW

**Comment by rawiamoalla** [comment]
Could you please merge this merge request

File: src/canmatrix/formats/fibex.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ def dump(db, f, **options):
     cluster.set('ID', 'canCluster1')
     create_short_name_desc(cluster, "clusterShort", "clusterDesc")
     create_sub_element_fx(cluster, "SPEED", "500")
-    create_sub_element_fx(cluster, "IS-HIGH-LOW-BIT-ORDER", "false")
```
Comment (ebroecker) [inline src/canmatrix/formats/fibex.py:104]: hi  @rawiamoalla 

are you sure here?  I did not double check, so just a question.

This is a change - the rest of your patch is no problem.

Comment (rawiamoalla) [reply src/canmatrix/formats/fibex.py:104]: Hi @ebroecker normally yes. there is no problem if we replaced to true

Comment (ebroecker) [reply src/canmatrix/formats/fibex.py:104]: ok, perfect

**Pull request merged**